azure-quantum/tests.live/Run.ps1

```diff
@@ -105,6 +105,10 @@ if ($PackageDir -Match "azure-quantum") {
     Copy-Item -Path (Join-Path $PackageDir "tests" "unit" "*.qs") -Destination $PSScriptRoot
 }
 
+# Pin urllib3 for compatibility with vcrpy
+Write-Host "##[info]Install version of urllib3 that is compatible with vcrpy"
+pip install urllib3==1.26.15
```

Comment:
Could this be in the requirements.txt file instead?

Reply:
Or some other requirements file that is used during build

Reply:
The requirements files seem to apply to not just test environments. This seemed to by the only test-specific place to include the pin.